assets/js/base/components/text-input/validated.js

```diff
@@ -107,8 +114,8 @@ const ValidatedTextInput = ( {
 			}
 			ref={ inputRef }
 			onChange={ ( val ) => {
-				hideValidationError( errorId );
 				onChange( val );
+				hideValidationError( errorId );
```

Comment:
I'm curious, why the change of sequence here?

Reply:
O, I think it was because in onBlur I made a change where something was called before the onBlur from props and I changed this to match, but I ended up reverting one and not the other. I will change it back; it's not related to this work.